internal/oauthserver/oauth_server.go

```diff
@@ -20,33 +21,28 @@ import (
 )
 
 const (
-	// sessionName is the name used for storing authorization session data.
+	// this is the cookie name.
+	// TODO: hardcoding this means that only one oauth flow can be in progress at a time
```

Comment:
:///

why not just do `uuid.New()` each time?

Reply:
To clarify this just means a user can't be actively signing in to two apps at the same time. Once they're signed in to one they can sign in to another. It's not that big of a deal. If you use a uuid we won't know what it is when we get the callback. We can use the state param because the oauth provider is supposed to include that in the callback. Can do in a follow up